lib/barr/blocks/hdd.rb

```diff
@@ -4,12 +4,13 @@ class Hdd < Block
       attr_reader :device
       def initialize opts={}
         super
+	@name = opts[:name]
         @device = opts[:device]
       end
 
       def update
         total, used, perc = sys_cmd.chomp.split(" ")
-        @output = "#{used} / #{total} (#{perc})"                                                              
+        @output = "#{@name}: #{used} / #{total} (#{perc})"
```

Comment:
Nice idea, though the functionality for this is already provided via the `icon` option.

`Barr::Blocks::Hdd.new device: "sda2", icon: "/dev/sda2: "`